centaur/src/main/resources/standardTestCases/drs_tests/drs_usa_hca_data_in_jdr.inputs

```diff
@@ -0,0 +1,4 @@
+{
+    "drs_usa_hca_data_in_jdr.file": "drs://jade.datarepo-dev.broadinstitute.org/v1_4122d403-8d70-4922-ab6e-7db13e44a9a5_aad1af32-1203-464d-a86d-d9a48e95b214",
```

Comment:
TOL do you know why there were 5 test input files before?

Reply:
I don't know for sure, but I suspect that's what was available at the time. The files were very interesting at the time and caused all sorts of edge cases throughout the Cromwell code:

- HCA used query strings that were dropped by Cromwell's string->URI->string parsing
- HCA data repo entries used an ambiguous `name` field that differed from the end of GCS object name (ex: effectively saying `my_file.txt` is stored at `gs://foo/bar.bin`)
- HCA entries at the time contained `!` in the file extensions that tripped up some wiring somewhere (can't remember specifics)
- other non "hello world" issues that would only pop up with Real™ HCA data

So having as much variety as possible was ideal.

When asked for new HCA data, the mentioned URL is [the only one](https://bvdp-saturn-dev.appspot.com/#workspaces/general-dev-billing-account/DRS%20Test%20Workspace%20-%20Dev/data) listed in the test workspace shared by DSP:

<img width="1947" alt="Screen Shot 2021-03-03 at 10 02 18 AM" src="https://user-images.githubusercontent.com/791985/109825393-c95f1880-7c07-11eb-9029-303a0e672936.png">